prov/verbs/src/ep_rdm/verbs_rdm_cm.c

```diff
@@ -615,21 +617,76 @@ ssize_t fi_ibv_rdm_conn_cleanup(struct fi_ibv_rdm_conn *conn)
 	return ret;
 }
 
+static int fi_ibv_rdm_poll_cq(struct fi_ibv_rdm_ep *ep)
```

Comment:
Not sure that I got it correctly. Do you mean to stay all as is and add the additional check on connection state in `fi_ibv_rdm_tagged_poll_send` (where postponed send queue are emptied)?
Also seems when we are handling WorkComp `fi_ibv_rdm_tagged_poll_send` and we face with !WC_STATUS_OK, other WC won't be handled (only !WC_STATUS_OK will be handled).